docs/configuration.rst

```diff
@@ -654,9 +689,27 @@ Map Options
     is required. If you want a high detail, use texture size 16, but texture 
     size 12 looks still good and is faster to render.
 
-    Mapcrafter's pre-rendered textures include sizes 16 and 12. If you want to 
+    Mapcrafter's pre-rendered spritemaps include sizes 16 and 12. If you want to 
     use other sizes, or custom resource packs you will need to generate them using 
-    `blockcrafter <https://github.com/mapcrafter/blockcrafter>`_.
+    `Blockcrafter <https://github.com/mapcrafter/blockcrafter>`_.
+
+
+**Block Directory** ``block_dir = <directory>``
+
+    **Default:** ``data/blocks``
+
+    This is the folder where spritemaps files are located. You can set this
```

Comment:
Spritemaps sounds nice, it's better understandable than block images perhaps. We should probably try to use that naming consistently, but I'd like to include somehow the word `blocks` in the name too. Maybe `block sprites`. I'll think about it.

Reply:
I've changes to *block sprites*, and sometimes just *sprites*. See what you think. I'm not 100% sold, but can't think of anything better.